src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -85,28 +87,48 @@ export const CloudEnvironmentModal = ({
       onDismiss()
     }
   })
+  const renderGalaxyModal = () => renderAppModal(GalaxyModalBase, NEW_GALAXY_MODE)
+  const renderCromwellModal = () => renderAppModal(CromwellModalBase, NEW_CROMWELL_MODE)
 
-  const renderDefaultPage = () => div({ style: { display: 'flex', flexDirection: 'column', flex: 1 } }, [
-    renderToolButtons(tools.Jupyter.label),
-    renderToolButtons(tools.RStudio.label),
-    renderToolButtons(tools.galaxy.label)
-  ])
+  const renderDefaultPage = () => div({ style: { display: 'flex', flexDirection: 'column', flex: 1 } },
+    _.map(tool => renderToolButtons(tool.label))(getToolsToDisplay)
+  )
 
   const toolPanelStyles = {
-    backgroundColor: 'white', margin: '0 1.5rem 1rem 1.5rem', padding: '0 1rem 1rem 1rem', display: 'flex', flexDirection: 'column'
+    backgroundColor: 'white',
+    margin: '0 1.5rem 1rem 1.5rem',
+    padding: '0 1rem 1rem 1rem',
+    display: 'flex',
+    flexDirection: 'column'
+  }
+  const toolLabelStyles = {
+    margin: '1rem 0 0.5rem 0',
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between'
   }
-  const toolLabelStyles = { margin: '1rem 0 0.5rem 0', display: 'flex', flexDirection: 'row', alignItems: 'center', justifyContent: 'space-between' }
   const toolButtonDivStyles = { display: 'flex', flexDirection: 'row', justifyContent: 'space-evenly' }
   const toolButtonStyles = {
-    flex: '1 1 0%', maxWidth: 105, display: 'flex', flexDirection: 'column', border: '.5px solid grey', borderRadius: 16, padding: '.5rem .75rem',
-    alignItems: 'center', fontWeight: 550, fontSize: 11, color: colors.accent()
+    flex: '1 1 0%',
+    maxWidth: 105,
+    display: 'flex',
+    flexDirection: 'column',
+    border: '.5px solid grey',
+    borderRadius: 16,
+    padding: '.5rem .75rem',
+    alignItems: 'center',
+    fontWeight: 550,
+    fontSize: 11,
+    color: colors.accent()
   }
 
   const currentRuntime = getCurrentRuntime(runtimes)
   const currentRuntimeStatus = getConvertedRuntimeStatus(currentRuntime)
   const currentRuntimeTool = currentRuntime?.labels?.tool
 
-  const currentApp = getCurrentApp(apps)
+  const currentApp = toolLabel => getCurrentAppForType(getAppType(toolLabel))(apps)
+  const showPause = toolLabel => !_.find(tool => tool.label === toolLabel)(tools).isPauseUnsupported
```

Comment:
OK, switching to isPauseSupported for this variable.

(I only made the method on the tools object a isPauseUnsupported because I needed the default value to be False).